addons/product/models/product_template.py

```diff
@@ -149,6 +149,10 @@ def _read_group_categ_id(self, categories, domain):
     default_code = fields.Char(
         'Internal Reference', compute='_compute_default_code',
         inverse='_set_default_code', store=True)
+    hs_code = fields.Char(
```

Comment:
I'm not sure we want to display this field all the time for everyone since it's only for customs...